ChangeLog.md

```diff
@@ -15,7 +15,7 @@
         - Added support for querying using Scope '/', '/apis', '/apis/echo-api'
 * Fix for issue https://github.com/Azure/azure-powershell/issues/9307 and https://github.com/Azure/azure-powershell/issues/8432
     - **Import-AzApiManagementApi**
-        - Added support for specifiying 'ApiVersion' and 'ApiVersionSetId' when importing Apis
```

Comment:
Do not make type corrections to release notes in existing psd1 files, code comments, or existing changelog comments